.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+sudo: required
+services: docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+matrix:
+  include:
+  - env: ELASTIC_STACK_VERSION=5.6.13
+  - env: ELASTIC_STACK_VERSION=6.4.3
+  - env: ELASTIC_STACK_VERSION=7.0.0-alpha1-SNAPSHOT
```

Comment:
I know I've asked this question in another PR but hardcoding versions will not scale, when the stack is upgraded we will have to update all plugins `.travis.yml`. Isn't there  a way for us to dynamically inject a matrix somehow which can be maintained and updated externally?